domains/runtime/evm/src/lib.rs

```diff
@@ -706,9 +727,11 @@ impl pallet_evm::Config for Runtime {
     type OnCreate = ();
     type FindAuthor = FindAuthorTruncated;
     type GasLimitPovSizeRatio = GasLimitPovSizeRatio;
-    type SuicideQuickClearLimit = SuicideQuickClearLimit;
     type Timestamp = Timestamp;
     type WeightInfo = pallet_evm::weights::SubstrateWeight<Self>;
+    type AccountProvider = pallet_evm::FrameSystemAccountProvider<Self>;
+    // TODO: re-check this value mostly from moonbeam
+    type GasLimitStorageGrowthRatio = ();
```

Comment:
Here is how it's implemented in the `pallet-evm` template:
https://github.com/polkadot-evm/frontier/blob/5a3f891485031dbcf41f2d3a2cd6bb45c18db4c1/template/runtime/src/lib.rs#L349

We might want to use some reasonable default here, because `()` means `0`, which gives unlimited storage usage during contract deployment. An "out of gas" error might be better if someone mistakenly deploys a contract with large storage usage.
https://github.com/polkadot-evm/frontier/blob/5a3f891485031dbcf41f2d3a2cd6bb45c18db4c1/frame/evm/src/runner/stack.rs#L1101-L1108